data/profiles/solarstorm2012.profile

```diff
@@ -0,0 +1,124 @@
+{
+  "actions": [
+    {
+      "documentation": "Reset button. Sets time to start at start of dataset. No loop",
+      "gui_path": "2012July",
+      "identifier": "2012July.Reset-loop",
+      "is_local": false,
+      "name": "Reset button.",
+      "script": "openspace.time.setTime('2012-JUL-01 07:00:00.00');\nopenspace.scriptScheduler.clear();"
+    },
+    {
+      "documentation": "Makes the sun turn black",
+      "gui_path": "2012July",
+      "identifier": "2012July.Dark-sun",
+      "is_local": false,
+      "name": "Dark sun",
+      "script": "openspace.setPropertyValueSingle('Scene.Sun.Renderable.Layers.ColorLayers.Texture.Settings.Multiplier', 0.000000);"
+    },
+    {
+      "documentation": "Sets time to start of data. sets high delta time. loops from start when at end of data",
+      "gui_path": "2012July",
+      "identifier": "2012July.loopEnlil",
+      "is_local": false,
+      "name": "Loop ENLIL",
+      "script": "openspace.scriptScheduler.clear();\nopenspace.time.setDeltaTime(43200);\nopenspace.time.setTime('2012-JUL-01 07:00:00.00');\nStarttimescriptENLILLoop = \"openspace.time.setTime('2012 JUL 01 07:00:00')\";\nopenspace.scriptScheduler.loadScheduledScript('2012 AUG 01 06:58:00', StarttimescriptENLILLoop);"
+    },
+    {
+      "documentation": "Sets time to start of data. Sets higher delta time. Loops back to start, at end of data.",
+      "gui_path": "2012July",
+      "identifier": "2012July.LoopBatsrus",
+      "is_local": false,
+      "name": "Loop BATSRUS",
+      "script": "openspace.scriptScheduler.clear();\nopenspace.time.setDeltaTime(1400);\nopenspace.time.setTime('2012-JUL-14 06:00:00.00');\nStarttimescriptBatsrusLoop = \"openspace.time.setTime('2012 JUL 14 06:00:00')\";\nopenspace.scriptScheduler.loadScheduledScript('2012 JUL 16 07:30:00', StarttimescriptBatsrusLoop);"
```

Comment:
Same argument as above for using `local` for the variable definition to make the profile future proof for a strict Lua state